spec/controllers/api/v1/games_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::GamesController, type: :controller do
+  let!(:first_game) { Game.create(name: "Arkham Horror", min_player_count: 1, max_player_count: 8, description: "Horror game where you can punch Cthulhu in the face") }
```

Comment:
you should use `create!` here since you are assuming that the game is going to successfully insert into the database.